### DISCUSSION TITLE
Vercel dev command is faster than next dev?

### DISCUSSION THREAD

**Discussion by anabel29** [vercel/vercel#7308] [Help]
Hello!, first time here ✋ 

I have a Nextjs application, and was using `next dev` to run my app locally, but I tried `vercel dev` and it compiles faster...!(14seconds with `next dev`, 4 seconds with `vercel dev`). At first time, `vercel dev` took 14seconds, but when I use it again, tooks 4 seconds every time.

In the docs, its recommended to use `next dev` if you have a Next.js application, but when I run `vercel dev` looks like is running `next dev` behind, this is the output of my console when running `vercel dev`:
<img width="474" alt="Captura de Pantalla 2022-01-14 a la(s) 13 11 24" src="https://user-images.githubusercontent.com/53570384/149548152-3ebf3a37-ec18-4278-80e7-40261f78d220.png">

So my question here is: should I use `vercel dev `instead of `next dev` ... and why is faster than `next dev`?

**Comment by leerob** [comment]
Is there a GitHub repo we can reproduce this with?
